### PR TITLE
cp-148: Remove loader from button

### PR DESCRIPTION
**Pull request by Ekit18**
image

### CODE REVIEW

**Review by OlenaYatseniuk — COMMENTED** [review]
I think it is not a good idea to delete loading from all buttons. You can make this optional for buttons on the survey page, but not delete it at all.

File: frontend/src/libs/components/button/button.tsx [hunk 1]
```diff
@@ -5,15 +5,17 @@ type Properties = {
...
 
 const Button: React.FC<Properties> = ({
   type = 'button',
   label,
   style = 'primary',
   isLoading = false,
+  isDisabled = false,
 }) => (
-  <button type={type} className={styles[style]} disabled={isLoading}>
+  <button type={type} className={styles[style]} disabled={isDisabled}>
```
Comment (liza-veis) [inline frontend/src/libs/components/button/button.tsx]: If we pass isLoading it should be disabled automatically
```suggestion
  <button type={type} className={styles[style]} disabled={isDisabled || isLoading}>
```

**Review by liza-veis — CHANGES_REQUESTED** [review]

**Review by OlenaYatseniuk — DISMISSED** [review]

File: frontend/src/libs/components/button/button.tsx (hunk 1 above)
Comment (Ekit18) [reply frontend/src/libs/components/button/button.tsx]: done

File: frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx [hunk 2]
```diff
@@ -53,7 +53,12 @@ const SignInForm: React.FC<Properties> = ({ onSubmit }) => {
         />
 
         <div className={styles['submit']}>
-          <Button type="submit" label="Sign in" isLoading={isLoading} />
+          <Button
+            type="submit"
+            label="Sign in"
+            isLoading={isLoading}
+            isDisabled={isLoading}
```
Comment (liza-veis) [inline frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx]:
```suggestion
```

File: frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx [hunk 3]
```diff
@@ -61,7 +61,12 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }) => {
           errors={errors}
         />
         <div className={styles['submit-button']}>
-          <Button type="submit" label="Sign up" isLoading={isLoading} />
+          <Button
+            type="submit"
+            label="Sign up"
+            isLoading={isLoading}
+            isDisabled={isLoading}
```
Comment (liza-veis) [inline frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx]:
```suggestion
```

**Review by liza-veis — CHANGES_REQUESTED** [review]

File: frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx (hunk 3 above)
Comment (Ekit18) [reply frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx]: done

File: frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx (hunk 2 above)
Comment (Ekit18) [reply frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx]: done

**Review by fshabanov — APPROVED** [review]

**Review by liza-veis — APPROVED** [review]

**Pull request merged**